include/cocaine/hpack/msgpack_traits.hpp

```diff
@@ -92,7 +93,7 @@ struct msgpack_traits {
             if(source.via.u64 >= table.size() || source.via.u64 == 0) {
                 throw std::system_error(
                     std::make_error_code(std::errc::invalid_argument),
-                    "Invalid index for header table"
```

Comment:
You can replace `EINVAL` system errors with `cocaine::error_t`, but I'll do it myself.